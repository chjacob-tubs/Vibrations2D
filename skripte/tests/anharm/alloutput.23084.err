--- PyADF *** new job *** PyADF ---
 ridft ended normally
