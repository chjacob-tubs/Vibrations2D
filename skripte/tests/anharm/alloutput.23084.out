--- PyADF *** new job *** PyADF ---
 operating system is UNIX !
 hostname is aluminium

 ridft (aluminium) : TURBOMOLE V7.1 ( 20661 ) 10 Jun 2016 at 08:56:48
 Copyright (C) 2016 TURBOMOLE GmbH, Karlsruhe


    2020-12-01 10:38:29.192 



                                  r i d f t

                        DFT program with RI approximation 
                                for coulomb part 




                                                 
                                 References:     
                                                 
          TURBOMOLE:                             
              R. Ahlrichs, M. Baer, M. Haeser, H. Horn, and
              C. Koelmel
              Electronic structure calculations on workstation
              computers: the program system TURBOMOLE
              Chem. Phys. Lett. 162: 165 (1989)
          Density Functional:                              
              O. Treutler and R. Ahlrichs                      
              Efficient Molecular Numerical Integration Schemes
              J. Chem. Phys. 102: 346 (1995)                   
          Parallel Version:                                
              Performance of parallel TURBOMOLE for Density    
              Functional Calculations                          
              M. v. Arnim and R. Ahlrichs                      
              J. Comp. Chem. 19: 1746 (1998)                   
          RI-J Method:                                     
              Auxiliary Basis Sets to approximate Coulomb      
              Potentials                                       
              Chem. Phys. Lett. 240: 283 (1995)                
              K. Eichkorn, O. Treutler, H. Oehm, M. Haeser     
              and R. Ahlrichs                                  
              Chem. Phys. Lett. 242: 652 (1995)                
                                                           
              Auxiliary Basis Sets for Main Row Atoms and their
              Use to approximate Coulomb Potentials            
              K. Eichkorn, F. Weigend, O. Treutler and         
              R. Ahlrichs                                      
              Theo. Chem. Acc. 97: 119 (1997)                   
                                                           
              Accurate Coulomb-fitting basis sets for H to Rn 
              F. Weigend                                        
              Phys. Chem. Chem. Phys. 8: 1057 (2006)            
                                                           
          Multipole accelerated RI-J (MARI-J):             
              Fast evaluation of the Coulomb potential for     
              electron densities using multipole accelerated   
              resolution of identity approximation             
              M. Sierka, A. Hogekamp and R. Ahlrichs           
              J. Chem. Phys. 118: 9136 (2003)                  
          RI-JK Method:                                     
              A fully direct RI-HF algorithm: Implementation,
              optimised auxiliary basis sets, demonstration of
              accuracy and efficiency                         
              F. Weigend                                      
              Phys. Chem. Chem. Phys. 4: 4285 (2002)           
          Two-component HF and DFT with spin-orbit coupling:  
              Self-consistent treatment of spin-orbit         
              interactions with efficient Hartree-Fock and    
              density functional methods                      
              M. K. Armbruster, F. Weigend, C. van WÃ¼llen and 
              W. Klopper                                      
              Phys. Chem. Chem. Phys. 10: 1748 (2008)         
          Relativistic all-electron 2c calculations           
              An efficient implementation of two-component    
              relativistic exact-decoupling methods for large 
              molecules                                       
              D. Peng, N. Middendorf, F. Weigend, M. Reiher   
              J. Chem. Phys. 138, 184105 (2013)               
                                         


 


              +--------------------------------------------------+
              |      general information about current run       |
              +--------------------------------------------------+

 
 B-P86 functional
 exchange:    LDA + Becke (B88)
 correlation: LDA (VWN) + Perdew (P86)
 A DFT calculation using the RI-J approximation will be carried out.
 Allocatable memory for RI due to $ricore (MB):        2000


              +--------------------------------------------------+
              | Atomic coordinate, charge and isotop information |
              +--------------------------------------------------+

                    atomic coordinates            atom    charge  isotop
         -5.60454702   -0.29035977    0.05364957    o      8.000     0
         -5.29194212   -0.19535734   -1.75974369    h      1.000     0
         -6.74903631   -1.72932541    0.17549160    h      1.000     0
 
       center of nuclear mass  :   -5.65109150   -0.36555538   -0.04099382
       center of nuclear charge:   -5.68773546   -0.42475609   -0.11550555

              +--------------------------------------------------+
              |               basis set information              |
              +--------------------------------------------------+

              we will work with the 1s 3p 5d 7f 9g ... basis set
              ...i.e. with spherical basis functions...

   type   atoms  prim   cont   basis
   ---------------------------------------------------------------------------
    o        1     24     14   def2-SVP   [3s2p1d|7s4p1d]
    h        2      7      5   def2-SVP   [2s1p|4s1p]
   ---------------------------------------------------------------------------
   total:    3     38     24
   ---------------------------------------------------------------------------

   total number of primitive shells          :   17
   total number of contracted shells         :   12
   total number of cartesian basis functions :   25
   total number of SCF-basis functions       :   24


 integral neglect threshold       :  0.13E-08
 integral storage threshold THIZE :  0.10E-04
 integral storage threshold THIME :         5

 RI-J AUXILIARY BASIS SET information:

              we will work with the 1s 3p 5d 7f 9g ... basis set
              ...i.e. with spherical basis functions...

   type   atoms  prim   cont   basis
   ---------------------------------------------------------------------------
    o        1     70     49   def2-SVP   [6s4p3d1f1g|12s5p4d2f1g]
    h        2     16     11   def2-SVP   [3s1p1d|5s2p1d]
   ---------------------------------------------------------------------------
   total:    3    102     71
   ---------------------------------------------------------------------------

   total number of primitive shells          :   32
   total number of contracted shells         :   25
   total number of cartesian basis functions :   85
   total number of SCF-basis functions       :   71


 symmetry group of the molecule :   c1 

 the group has the following generators :
   c1(z)

    1 symmetry operations found

 there are 1 real representations :   a   

 maximum number of shells which are related by symmetry :  1


    mo occupation :
   irrep   mo's   occupied
    a       24        5
 
 number of basis functions   :           24
 number of occupied orbitals :            5
 
  
           ------------------
           density functional
           ------------------
 B-P86 functional
 exchange:    LDA + Becke (B88)
 correlation: LDA (VWN) + Perdew (P86)
 iterations will be done with small grid
  
 spherical integration : Lebedev's spherical grid
 spherical gridsize    :           4
    i.e. gridpoints    :         434
 value for diffuse not defined
 radial integration    : Chebyshev 2nd kind (scaling 3)
 radial gridsize       :           6
 integration cells     :           3
 partition function    : becke
 partition sharpness   :           3
  

 biggest AO integral is expected to be     4.776656448
  
           ------------------------
               RI-J - INFORMATION
           ------------------------
 Contributions to RI integral batches: 
  neglected integral batches:           0
  direct contribution:                  0
  memory contribution:          78
 Memory core needed for (P|Q) and Cholesky      1 MByte
 Memory core minimum needed except of (P|Q)     1 MByte
 Total minimum memory core needed (sum)         1 MByte
  
 ****************************************
 Memory allocated for RI-J     1 MByte
 ****************************************
                                            

          ------------------------
          nuclear repulsion energy  :   9.03206996349    
          ------------------------

 
          -----------------
          -S,T+V- integrals
          -----------------

 1e-integrals will be neglected if expon. factor < 0.133678E-09
 
   Difference densities algorithm switched on.
   The maximal number of linear combinations of
   difference densities is          20 .

 DIIS switched on: error vector is FDS-SDF
 Max. Iterations for DIIS is     :   4
 DIIS matrix (see manual) 
    Scaling factor of diagonals  :  1.200
    threshold for scaling factor :  0.000

 scf convergence criterion : increment of total energy < .1000000D-05
                  and increment of one-electron energy < .1000000D-02


 <geterg> : there is no data group $energy 

  MOs are in ASCII format !


 automatic virtual orbital shift switched on 
      shift if e(lumo)-e(homo) < 0.10000000    


 reading orbital data $scfmo  from file mos
 orbital characterization : expanded
 mo provided and orthogonalized by Cholesky decomposition

 DSCF restart information will be dumped onto file mos


 Starting SCF iterations

          Overall gridpoints after grid construction =          7368

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   1  -76.143366268139    -124.11195108     38.936514853    0.000D+00 0.133D-08
                            Exc = -9.49905363610     Coul =  48.4355684890    
                              N = 9.9999832947    
                            current damping = 0.700
 
          max. resid. norm for Fia-block=  2.824D-01 for orbital      4a         
          max. resid. fock norm         =  1.281D+00 for orbital     18a         

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   2  -76.323967451236    -122.80278477     37.446747356    0.450D+00 0.881D-09
                            Exc = -9.31060604865     Coul =  46.7573534042    
                              N = 9.9999834402    
                            current damping = 0.650
 
          Norm of current diis error: 0.47815    
          max. resid. norm for Fia-block=  1.201D-01 for orbital      1a         
          max. resid. fock norm         =  1.287D-01 for orbital      6a         

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   3  -76.354007090567    -123.00749059     37.621413539    0.177D+00 0.667D-09
                            Exc = -9.35617343108     Coul =  46.9775869699    
                              N = 9.9999830095    
                            current damping = 0.600
 
          Norm of current diis error: 0.22886    
          max. resid. norm for Fia-block=  4.722D-02 for orbital      1a         
          max. resid. fock norm         =  5.127D-02 for orbital      6a         

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   4  -76.362191237270    -122.75134676     37.357085556    0.623D-01 0.575D-09
                            Exc = -9.33182178274     Coul =  46.6889073388    
                              N = 9.9999826519    
                            current damping = 0.650
 
          Norm of current diis error: 0.77214E-01
          max. resid. norm for Fia-block=  1.816D-02 for orbital      1a         
          max. resid. fock norm         =  2.193D-02 for orbital      6a         

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   5  -76.363341165463    -122.81343458     37.418023456    0.133D-01 0.530D-09
                            Exc = -9.34482685582     Coul =  46.7628503114    
                              N = 9.9999825529    
                            current damping = 0.700
 
          Norm of current diis error: 0.31943E-01
          max. resid. norm for Fia-block=  7.042D-03 for orbital      1a         
          max. resid. fock norm         =  8.380D-03 for orbital      6a         
          mo-orthogonalization: Cholesky decomposition

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   6  -76.363544188121    -122.78887889     37.393264734    0.190D-02 0.501D-09
                            Exc = -9.34329334155     Coul =  46.7365580760    
                              N = 9.9999825004    
                            current damping = 0.750
 
          Norm of current diis error: 0.12354E-01
          max. resid. norm for Fia-block=  2.935D-03 for orbital      1a         
          max. resid. fock norm         =  3.594D-03 for orbital      6a         

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   7  -76.363576030519    -122.79271683     37.397070834    0.242D-03 0.482D-09
                            Exc = -9.34452175445     Coul =  46.7415925889    
                              N = 9.9999824811    
                            current damping = 0.800
 
          Norm of current diis error: 0.53601E-02
          max. resid. norm for Fia-block=  1.243D-03 for orbital      1a         
          max. resid. fock norm         =  1.517D-03 for orbital      6a         

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   8  -76.363582058381    -122.79083831     37.395186286    0.411D-04 0.469D-09
                            Exc = -9.34456740616     Coul =  46.7397536918    
                              N = 9.9999824726    
                            current damping = 0.850
 
          Norm of current diis error: 0.23431E-02
          max. resid. norm for Fia-block=  5.584D-04 for orbital      1a         
          max. resid. fock norm         =  6.951D-04 for orbital      6a         

 ENERGY CONVERGED !

          Overall gridpoints after grid construction =         28308

 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     NORM[dD(SAO)]  TOL
   9  -76.363581864193    -122.79198319     37.396331366    0.710D-05 0.458D-09
                            Exc = -9.34484684234     Coul =  46.7411782084    
                              N = 10.000000041    
                            current damping = 0.900
 
          Norm of current diis error: 0.11078E-02
          max. resid. norm for Fia-block=  2.530D-04 for orbital      1a         
          max. resid. fock norm         =  3.095D-04 for orbital      6a         

 End of SCF iterations

   convergence criteria satisfied after    9 iterations


                  ------------------------------------------ 
                 |  total energy      =    -76.36358186419  |
                  ------------------------------------------ 
                 :  kinetic energy    =     75.78527611405  :
                 :  potential energy  =   -152.14885797824  :
                 :  virial theorem    =      1.99242694311  :
                 :  wavefunction norm =      1.00000000000  :
                  .......................................... 


 <geterg> : there is no data group $energy 


 <skperg> : $end is missing 


 orbitals $scfmo  will be written to file mos

    irrep                  1a          2a          3a          4a          5a   
 eigenvalues H        -18.76161    -0.89089    -0.45529    -0.31344    -0.23300
            eV        -510.5334    -24.2426    -12.3891     -8.5293     -6.3403
 occupation              2.0000      2.0000      2.0000      2.0000      2.0000 

    irrep                  6a          7a          8a          9a         10a   
 eigenvalues H          0.02949     0.10577     0.50740     0.57119     0.86216
            eV           0.8026      2.8783     13.8071     15.5429     23.4607
 
 
 
 
 ==============================================================================
                           electrostatic moments
 ==============================================================================

 reference point for electrostatic moments:    0.00000   0.00000   0.00000

 
              nuc           elec       ->  total
 ------------------------------------------------------------------------------
                          charge      
 ------------------------------------------------------------------------------
          10.000000     -10.000000      -0.000000
 
 ------------------------------------------------------------------------------
                       dipole moment  
 ------------------------------------------------------------------------------
   x     -56.877355      56.596349      -0.281005
   y      -4.247561       3.793563      -0.453998
   z      -1.155055       0.583635      -0.571421
 
   | dipole moment | =     0.7820 a.u. =     1.9878 debye 
 
 ------------------------------------------------------------------------------
                     quadrupole moment
 ------------------------------------------------------------------------------
  xx     324.841721    -326.259975      -1.418254
  yy       3.703201      -7.708954      -4.005753
  zz       3.150521      -6.919818      -3.769296
  xy      25.723779     -22.322585       3.401194
  xz       5.722610      -2.895664       2.826946
  yz      -0.084325       0.019854      -0.064471
 
     1/3  trace=      -3.064435
     anisotropy=       8.051765
 
 ==============================================================================
 


    ------------------------------------------------------------------------
         total  cpu-time :   0.21 seconds
         total wall-time :   0.99 seconds
    ------------------------------------------------------------------------

   ****  ridft : all done  ****


    2020-12-01 10:38:30.105 

